internal/db/migrations/2023-01-27.4-create-messages-table.sql

```diff
@@ -20,6 +20,34 @@ CREATE TABLE public.messages (
     created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
 );
 
+-- +migrate StatementBegin
+-- Insert messages if there is pre-existing data, so to prevent SMSs from being re-triggered.
+DO $$
+BEGIN
+    IF (SELECT COUNT(*) FROM public.receivers WHERE status <> ALL(ARRAY['READY', 'PAID', 'PARTIALLY_CASHED_OUT', 'FULLY_CASHED_OUT'])) > 0 THEN 
+        INSERT INTO public.messages (
+            type,
+            asset_id,
+            wallet_id,
+            receiver_id,
+            text_encrypted,
+            title_encrypted,
+            created_at
+        )
+        SELECT 
+            'AWS_SMS',
+            (SELECT id FROM public.assets WHERE code = 'USDC' LIMIT 1),
+            (SELECT id FROM public.wallets WHERE name ILIKE '%Vibrant%' LIMIT 1),
```

Comment:
FYI, in `wallets_pubnet.go`, we have `Vibrant Assist` and `Vibrant Assist RC`. Not sure if it matters here.

Reply:
Thanks for highlighting that. I've added an instruction to the migration doc that ensures this wont be a problem.